linkerd/dns/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "linkerd2-dns"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+futures = "0.1"
+linkerd2-dns-name = { path = "./name" }
+# FIXME update to a release when available (>0.11)
+trust-dns-resolver = { git = "https://github.com/bluejekyll/trust-dns", rev = "7c8a0739dad495bf5a4fddfe86b8bbe2aa52d060", default-features = false }
```

Comment:
FWIW, the release the `FIXME` mentions has been out for a while: https://crates.io/crates/trust-dns-resolver/0.11.1

(fine with updating that in a separate PR)